### PR TITLE
Update holiday information based on recent questions

### PR DESCRIPTION
**Pull request by lukemorton**
This change hopefully makes the information a bit easier to digest and also puts more onus on the staff member to use their initiative.

### CODE REVIEW

File: benefits/untracked_holiday.md [hunk 1]
```diff
@@ -4,22 +4,32 @@ We're building a high-performance organisation that delivers great software prod
...
-* If you're only taking a few days off, this can be approved by one of your sprint or support team mates.
+## How long are you taking?
 
-* If you're taking more than one week off, this should be approved by Chris or Rory at least one month in advance. 
+We trust you'll use your best judgement when taking holidays but in case you want some guidance on the kind of notice we expect when you take your holiday:
 
-* Having several team members away at the same time can make life really difficult for everyone else. Try to forward-plan your holiday, so it doesn't have a detrimental affect on your team mates or on the company.
+* **Less than 1 week:**<br>We recommend giving at least 2 weeks notice for a 1 day off and closer to 4 weeks notice for more
+* **Between 1 and 2 weeks:**<br>We recommend giving at least 4 weeks notice for this kind of holiday, ideally more.
+* **Greater than 3 weeks:**<br>Please try and provide at least three months notice, so we can prepare for your absence.
```
Comment (chrisblackburn) [inline benefits/untracked_holiday.md]: Try *to* provide

File: benefits/untracked_holiday.md [hunk 2]
```diff
@@ -4,22 +4,32 @@ We're building a high-performance organisation that delivers great software prod
...
+
+## Additional considerations
 
-* If you're taking three (or more) weeks off, please try and provide at least three months notice, so we can prepare for your absence. 
+* We use the calendar when working out project allocation and the number of days being billed to customers, so it's important that it is always up-to-date and accurate.
 
 * Employees will not “accrue” holiday days under this policy, therefore you will not receive compensation for “unused” holiday time when you leave.
```
Comment (chrisblackburn) [inline benefits/untracked_holiday.md]: Contractual obligations don't exactly mirror the policy in practice, so this isn't strictly true

Comment (lukemorton) [inline benefits/untracked_holiday.md]: I'll just remove it then?

File: benefits/untracked_holiday.md [hunk 3]
```diff
@@ -4,22 +4,32 @@ We're building a high-performance organisation that delivers great software prod
...
 We have adopted an untracked holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. 
 
-All we ask is that you follow a few guidelines:
+## Who approves holiday?
 
-* We use a shared Google Calendar to track who's away on holiday. Once booked, please check that your holiday is recorded in here. This calendar is used when working out project allocation and the number of days being billed to customers, so it's important that it is always up-to-date and accurate. [View our holiday calendar](https://www.google.com/calendar/ical/madebymade.co.uk_83gtf1a3s2ecshli2o6ub4kpmk%40group.calendar.google.com/public/basic.ics)
+We want to ensure teams own their holidays and that includes thinking about the commercial impact of taking them. To that affect we need to make sure you've done the following before booking holiday:
 
-* To book holiday, just send an email to the most appropriate person and include the dates of that you require.
+* Ask your team mates if it's okay with them
```
Comment (chrisblackburn) [inline benefits/untracked_holiday.md]: I think teammate can be one word

Comment (chrisblackburn) [inline benefits/untracked_holiday.md]: (depending on whether you want it to be)

File: benefits/untracked_holiday.md (hunk 2 above)
Comment (chrisblackburn) [inline benefits/untracked_holiday.md]: I think so, yep

**Comment by lukemorton** [comment]
@chrisblackburn Have addressed feedback. All good?

**Comment by RyanMacG** [comment]
@lukemorton can't see anything more than the feedback Chris has given already

**Comment by craigjbass** [comment]
Highly approve of this policy,

**Comment by craigjbass** [comment]
Lol, wrong button

**Comment by lukemorton** [comment]
Good work @craigjbass

**Comment by craigjbass** [comment]
It's a very Teal policy. With a healthy amount of recommendations (creating team norms)

File: benefits/untracked_holiday.md [hunk 4]
```diff
@@ -4,22 +4,30 @@ We're building a high-performance organisation that delivers great software prod
...
+* **Between 1 and 2 weeks**<br>We recommend giving at least 4 weeks notice for this kind of holiday, ideally more.
+* **Greater than 3 weeks**<br>Please try to provide at least three months notice, so we can prepare for your absence.
 
-* If you're taking three (or more) weeks off, please try and provide at least three months notice, so we can prepare for your absence. 
+## Additional considerations
 
-* Employees will not “accrue” holiday days under this policy, therefore you will not receive compensation for “unused” holiday time when you leave.  
+* We use the calendar when working out project allocation and the number of days being billed to customers, so it's important that it is always up-to-date and accurate.
 
 * Time off due to illness and/or injury are handled separately. Unlimited holiday cannot be used for these types of absences.
```
Comment (RoryMacDonald) [inline benefits/untracked_holiday.md]: Unlimited to Untracked

Comment (lukemorton) [inline benefits/untracked_holiday.md]: Done direct to master, cheers Rory.

**Pull request merged**